### PR TITLE
CNDIT 1445 + CNDIT 1421

### PR DESCRIPTION
**Pull request by upasanapattnaik-eq**
Bug Fix [CNDIT-1445](https://cdc-nbs.atlassian.net/browse/CNDIT-1445) and [CNDIT-1421](url): Updates to investigation post-processing stored procedure and nrt_investigation table schema to accommodate the format of legacy_case_ids, which according to the case notification mapping guide, can be alphanumeric- `Caseld01LAB2009`, and correctly map EARLIEST_RPT_TO_STATE_DT column.

Updates to Liquibase scripts:
- nrt_investigation
- sp_nrt_investigation_postprocessing:

### CODE REVIEW

File: data-reporting-service/src/main/resources/db/rdb/023-sp_nrt_investigation_postprocessing-001.sql [hunk 1]
```diff
@@ -1,501 +1,643 @@
...
-             left join dbo.investigation i on i.case_uid = nrt.public_health_case_uid
-    where nrt.public_health_case_uid in (select value FROM STRING_SPLIT(@id_list, ','));
-
-
-    -- if confirmation_method_key for the cd exists get the key or insert a new row to rdb.confirmation_method
-
-    /*Update Operation for confirmation_method and group*/
-    update cm
-    set cm.CONFIRMATION_METHOD_DESC = cmt.CONFIRMATION_METHOD_DESC_TXT
+	/* Logging */
```
Comment (alog-enquizit) [inline data-reporting-service/src/main/resources/db/rdb/023-sp_nrt_investigation_postprocessing-001.sql:7]: is this due to the formatting change

File: data-reporting-service/src/main/resources/db/rdb/008-create_nrt_investigation-001.sql [hunk 2]
```diff
@@ -79,7 +79,7 @@ CREATE TABLE dbo.nrt_investigation (
     curr_process_state            varchar(100)                                    NULL,
     inv_priority_cd               varchar(100)                                    NULL,
     coinfection_id                varchar(100)                                    NULL,
-    legacy_case_id                bigint                                          NULL,
+    legacy_case_id                varchar(100)                                    NULL,
```
Comment (alog-enquizit) [inline data-reporting-service/src/main/resources/db/rdb/008-create_nrt_investigation-001.sql:82]: 👍

**Review by alog-enquizit — APPROVED** [review]
lgtm 👍

File: data-reporting-service/src/main/resources/db/rdb/023-sp_nrt_investigation_postprocessing-001.sql (hunk 1 above)
Comment (upasanapattnaik-eq) [reply data-reporting-service/src/main/resources/db/rdb/023-sp_nrt_investigation_postprocessing-001.sql:7]: Yes :/ Let me remove the [] and push a new change.

File: data-reporting-service/src/main/resources/db/rdb/008-create_nrt_investigation-001.sql (hunk 2 above)
Comment (sveselev) [reply data-reporting-service/src/main/resources/db/rdb/008-create_nrt_investigation-001.sql:82]: LGTM!

**Review by sveselev — APPROVED** [review]

**Pull request merged**